packages/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -347,6 +347,7 @@ spec:
         - #@ "--metrics-bind-addr={}".format(data.values.tanzuAddonsManager.deployment.metricsBindAddress)
         - #@ "--leader-elect={}".format(data.values.tanzuAddonsManager.deployment.leaderElection)
         - #@ "--health-addr=:{}".format(data.values.tanzuAddonsManager.deployment.healthzPort)
+        - #@ "--cluster-delete-timeout=:{}".format(data.values.tanzuAddonsManager.deployment.clusterDeleteTimeout)
```

Comment:
I dont think there should be a colon (:) here. Thats only for health-addr since its refering to a addr:port

Reply:
done